### ISSUE TITLE
How to use this in an IDE?

### ISSUE THREAD

**Issue by aliakhtar** [deepseek-ai/DeepSeek-Coder#75]
Hi, is there a way to use this model inside an IDE, e.g Intellij?

**Comment by wwwzhouhui** [comment]
I have a plan to implement it using codegpt+DeepSeek-Coder

**Comment by aliakhtar** [comment]
@wwwzhouhui  any chance you could share an english version of that video please, or just describe how you integrated it with intellij?

It looks like you got text-generation-web-ui to run the model locally, then hooked that with codegpt?

**Comment by wwwzhouhui** [comment]
@aliakhtar Yes, the video mentioned that text-generation-webui integrates the codegpt project

**Comment by RobToMars** [comment]
I made use of the third-party OLLAMA integration possibility
[https://github.com/RobToMars/DeepSeek](https://github.com/RobToMars/DeepSeek)

**Comment by itdir** [comment]
> Hi, is there a way to use this model inside an IDE, e.g Intellij?

I don't understand how this doesn't have 1000s of likes. (It's the only reason I still use CodePilot.)
